bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/extensions/PersistenceExtensions.java

```diff
@@ -80,7 +80,7 @@ public PersistenceExtensions(@Reference PersistenceServiceRegistry registry,
      * @param item the item to store
      */
     public static void persist(Item item) {
-        internalPersist(item);
+        internalPersist(item, (String) null);
```

Comment:
Why do you need a cast to `String` here?

Reply:
Was needed when I only half changed the file (version with signature `internalPersist(Item, TimeSeries)`), but no longer needed, so removed.